code/game/machinery/_machinery.dm

```diff
@@ -568,6 +568,10 @@
 
 /obj/proc/default_unfasten_wrench(mob/user, obj/item/I, time = 20) //try to unwrench an object in a WONDERFUL DYNAMIC WAY
 	if(!(flags_1 & NODECONSTRUCT_1) && I.tool_behaviour == TOOL_WRENCH)
+		var/turf/ground = get_turf(src)
+		if(!anchored && ground.is_blocked_turf(TRUE, TRUE, src))
```

Comment:
Change this to use named parameters--`TRUE, TRUE` is not obvious what it's doing.